lib/spaceship/tunes/tunes_client.rb

```diff
@@ -776,19 +791,38 @@ def remove_tester_from_app!(tester, app_id)
 
     private
 
+    def with_tunes_retry(tries = 3, &block)
+      return block.call
+    rescue Spaceship::TunesClient::ITunesConnectTemporaryError => ex
+      unless (tries -= 1).zero?
+        msg = "ITC temporary save error received: '#{ex.message}'. Retrying after 60 seconds (remaining: #{tries})..."
```

Comment:
Do we have access to the `tries` variable here? If yes, we could start by waiting just like a few seconds, and then go up to 1-2 minutes